integration/spark-common-test/src/test/resources/2000_testCSV.csv

```diff
@@ -0,0 +1,18 @@
+,,,,,,,,,,,
```

Comment:
Is there any particular reason to add a new CSV file ? If not, can you use already existing CSV file ?

Reply:
yes, there issue is reported by tester. He use this csv file, it's difficult to replace it. I need reproduce it and validate with this file.

Reply:
Please avoid adding files if not required. We can reproduce the root cause with any sample data.

Reply:
done

Reply:
removed